OrbitClientModel/CaptureSerializerTest.cpp

```diff
@@ -129,6 +148,20 @@ TEST(CaptureSerializer, GenerateCaptureInfo) {
   EXPECT_EQ(callstack_event.time(), actual_callstack_event.time());
   EXPECT_EQ(callstack_event.callstack_hash(), actual_callstack_event.callstack_hash());
 
+  ASSERT_EQ(1, capture_info.tracepoint_infos_size());
+  const orbit_client_protos::TracepointInfo& actual_tracepoint_info =
+      capture_info.tracepoint_infos(0);
+  ASSERT_EQ(actual_tracepoint_info.category(), selected_tracepoint_info.category());
```

Comment:
Nit: These can be EXPECT_EQ rather then ASSERT_EQ.
The difference is, that the execution will be stopped, if an assert fails. On expect, only the failure will be reported, but the test continues.
This is why, the size checks are asserts, as we afterwards, access the data.

Reply:
Thank you so much! I will update the code!